viscm/gui.py

```diff
@@ -1050,7 +1053,7 @@ def main(argv):
     import signal
     signal.signal(signal.SIGINT, signal.SIG_DFL)
 
-    app.exec_()
+    (getattr(app, "exec", None) or getattr(app, "exec_"))()
```

Comment:
I think we can just `app.exec()` because we're dropping support for Qt4. What do you think?